### ISSUE TITLE
Spurious pre-update lifecycle hook errors being logged

### ISSUE THREAD

**Issue by ginkel** [containrrr/watchtower#380]
**Describe the bug**

When a container that does not have a label for a pre-update hook defined is to be updated, the following error is logged if `sh` is not included with said container:

```
time="2019-09-18T19:40:21Z" level=info msg="Executing pre-update command."
time="2019-09-18T19:40:22Z" level=error msg="Command exited with code 126."
time="2019-09-18T19:40:22Z" level=error msg="OCI runtime exec failed: exec failed: container_linux.go:345: starting container process caused \"exec: \\\"sh\\\": executable file not found in $PATH\": unknown"
```

The only Watchtower-related label defined is:

```
"com.centurylinklabs.watchtower.enable": "true"
```

**Expected behavior**
The pre-update command should not be executed if undefined.

**Environment**
 - Ubuntu 18.04
 - x86_64
 - Docker version 19.03.2, build 6a30dfc

**Logs from running watchtower with the `--debug` option**
Not available ATM

**Additional context**
Watchtower version:

```
REPOSITORY                         TAG                 IMAGE ID            CREATED             SIZE
containrrr/watchtower              latest              804de5d63c24        3 days ago          14.3MB
```

**Comment by simonaronsson** [comment]
Thanks for reporting! I’ll have a look as soon as possible.

**Comment by 0xERR0R** [comment]
I have the same problem. I figured out, this was already fixed with #373, commit 135467dc, but it was not in the last release. There was a missing "return" in the executePreUpdateCommand function.

**Comment by simonaronsson** [comment]
As @0xERR0R noted, this has already been fixed and will be included in the next release, which hopefully - will be out today.
